### DISCUSSION TITLE
GitHub Raids

### DISCUSSION THREAD

**Discussion by kaiwalyakoparkar** [EddieHubCommunity/support#1123] [Ideas] [answered]
As discussed in yesterday's live stream of @eddiejaoude. Should we be doing GitHub raids on the public repositories of our community members and help them gain value and make them more perfect?

**Comment by eddiejaoude** [comment, marked as answer]
Great idea 👍 

I think we do this a little when reviewing open source projects, but maybe we could do a more focused live stream with everyone adding value 💪
